packages/caliper-samples/network/scalardl/scalardl_simple.json

```diff
@@ -0,0 +1,43 @@
+{
+  "caliper" : {
+    "blockchain": "scalardl",
+    "command" : {
+      "start": "/bin/sh network/scalardl/scripts/up.sh docker-compose.yml 30",
+      "end" : "/bin/sh network/scalardl/scripts/down.sh docker-compose.yml"
+    }
+  },
+  "contract": { "path": "src/contract/scalardl" },
+  "scalardl": {
+    "network": {
+      "client_properties": {
+        "host": "localhost",
```

Comment:
Just a confirmation. If this is updated, we can check with a cluster with more nodes ?

Reply:
Yes. If the endpoint of the cluster is a single host, just replace the `localhost`.